src/lib/PasswordHasher.js

```diff
@@ -14,24 +14,37 @@
 *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *
 *    You can download the source here: https://github.com/spark/spark-server
+*
+* @flow
+*
 */
 
-var crypto = require('crypto');
-
-var PasswordHasher = function () {
+import crypto from 'crypto';
 
-};
+const HASH_DIGEST = 'sha1';
```

Comment:
Yeah.. so make sure that `sha1` was what they were using before.  There could be code in here that expects a certain algorithm.